docs/src/pages/components/cdk-components/link/api.jsx

```diff
@@ -63,6 +58,16 @@ const linkPropsTable = () => {
           be called when the user clicks the link.
         </td>
       </tr>
+      <tr>
+        <td>children: string | node</td>
+        <td></td>
+        <td>
+          Content to display in the link. To use the link with other libraries
```

Comment:
I think this description is a bit verbose, maybe reformulate it like: "Content of the link. To use the component with other libraries (such as React Router or Next.js Link) pass the custom link as a child. The component will apply the styles to the custom link.", what do you think?

Reply:
Yes, now is better 👍🏼